### PR TITLE
Update sc monitoring refs to 1.1.1

### PR DESCRIPTION
**Pull request by tmasternak**

### CODE REVIEW

File: src/ServiceControlInstaller.Packaging/ServiceControlInstaller.Packaging.csproj [hunk 1]
```diff
@@ -134,6 +134,7 @@
       <ZipTargetFolder>..\..\zip\</ZipTargetFolder>
       <ZipToCreate>$(ZipTargetFolder)Particular.ServiceControl-$(GitVersion_MajorMinorPatch).zip</ZipToCreate>
     </PropertyGroup>
+    <Message Text="$(ZipTargetFolder)Particular.ServiceControl-$(GitVersion_MajorMinorPatch).zip" Importance="high"/>
```
Comment (WilliamBZA) [inline src/ServiceControlInstaller.Packaging/ServiceControlInstaller.Packaging.csproj]: Does this have something to do with the build not completing?

Comment (WilliamBZA) [reply src/ServiceControlInstaller.Packaging/ServiceControlInstaller.Packaging.csproj]: Or is this a debugging test?

Comment (tmasternak) [reply src/ServiceControlInstaller.Packaging/ServiceControlInstaller.Packaging.csproj]: This is was (stupid I agree - the value outputed is already in the logs :P) debugging addition.

**Comment by tmasternak** [comment]
Based on https://github.com/gbiellem/DeploymentZipTask/pull/1 and https://github.com/gbiellem/DeploymentZipTask/pull/2

**Pull request merged**